mmcv/ops/correlation.py

```diff
@@ -15,14 +17,14 @@ class CorrelationFunction(Function):
 
     @staticmethod
     def forward(ctx,
-                input1,
-                input2,
-                kernel_size=1,
-                max_displacement=1,
-                stride=1,
-                padding=1,
-                dilation=1,
-                dilation_patch=1):
+                input1: Tensor,
+                input2: Tensor,
+                kernel_size: int = 1,
+                max_displacement: int = 1,
+                stride: int = 1,
```

Comment:
Should this stride be consistent with

Reply:
This `stride` should be consistent with the one in `class Correlation` at the same file, which is defined as `int`.

![Selection_001](https://user-images.githubusercontent.com/40317966/172097248-7728e449-0bd7-4273-b01b-3b7f08f5cfff.png)